packages/js/src/shared-admin/components/ai-generate-titles-and-descriptions-upsell.js

```diff
@@ -160,4 +165,5 @@ AiGenerateTitlesAndDescriptionsUpsell.defaultProps = {
 	newToText: "Yoast SEO Premium",
 	isProductCopy: false,
 	bundleNote: "",
+	ctbId: "f6a84663-465f-4cb5-8ba5-f7a6d72224b2",
```

Comment:
I think that should be an empty string.

```suggestion
	ctbId: "",
```

Reply:
I want this to be the default since this is the premium upsell so that if nothing is passed the upsell will always work.